### PR TITLE
python3Packages.wagtail-modeladmin: init at 2.0.0

### PR DESCRIPTION
**Pull request by sephii**
## Description of changes

https://github.com/wagtail-nest/wagtail-modeladmin

This package is recommended by wagtail-localize: https://github.com/wagtail/wagtail-localize/releases/tag/v1.7

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by pbsds** [comment]
Result of `nixpkgs-review pr 267816` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages failed to build:</summary>
  <ul>
    <li>python310Packages.wagtail-localize</li>
    <li>python310Packages.wagtail-localize.dist</li>
    <li>python310Packages.wagtail-modeladmin</li>
    <li>python310Packages.wagtail-modeladmin.dist</li>
    <li>python311Packages.wagtail-localize</li>
    <li>python311Packages.wagtail-localize.dist</li>
    <li>python311Packages.wagtail-modeladmin</li>
    <li>python311Packages.wagtail-modeladmin.dist</li>
  </ul>
</details>

```
error: builder for '/nix/store/hx0kzg7arswbjki923m08jrxkqyz9q4y-python3.10-wagtail-modeladmin-1.0.0.drv' failed with exit code 1;
       last 10 log lines:
       > Running phase: configurePhase
       > no configure script, doing nothing
       > Running phase: buildPhase
       > Executing pypaBuildPhase
       > Creating a wheel...
       > * Getting build dependencies for wheel...
       >
       > ERROR Missing dependencies:
       >        Wagtail>=5.0
       >        django-taggit<5.0,>=2.0
```

**Comment by sephii** [comment]
@pbsds should be fixed now.

File: pkgs/development/python-modules/wagtail-modeladmin/default.nix [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ];
+
+  pythonImportsCheck = [ "wagtail_modeladmin" ];
+
+  checkPhase = ''
+    ${python.interpreter} testmanage.py test
+  '';
+
+  meta = with lib; {
+    description = "Add any model in your project to the Wagtail admin. Formerly wagtail.contrib.modeladmin.";
```
Comment (pbsds) [inline pkgs/development/python-modules/wagtail-modeladmin/default.nix:50]:
```suggestion
    description = "Add any model in your project to the Wagtail admin. Formerly wagtail.contrib.modeladmin";
```

**Review by pbsds — COMMENTED** [review]
Result of `nixpkgs-review pr 267816` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>python311Packages.wagtail-localize</li>
    <li>python311Packages.wagtail-localize.dist</li>
    <li>python311Packages.wagtail-modeladmin</li>
    <li>python311Packages.wagtail-modeladmin.dist</li>
  </ul>
</details>

LGTM, with two exceptions

**Comment by SuperSandro2000** [comment]
@ofborg build python3Packages.wagtail-modeladmin

**Pull request merged**